README.md

```diff
@@ -9,3 +9,7 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+If two graphs have the same number of nodes then each node in graph A corresponds to a node in graph B, |V1| = |V2| = n, both sets had the same number of n (nodes). They don't have to be completely connected they just have to have the same number of edges and degree sequence for each node that corresponds with the node in the other graph. 
```

Comment:
But what about the second part of the definition? Matching nodes need to have the same edge connectivity.